### PR TITLE
P0 feature cart

### PR DESCRIPTION
**Pull request by swethaogeti**

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge it. One suggestion try to you use semantic tags.

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+               <hr>
+                    <p>You will save  <span>&#8377;999</span> on this order </p>
+                    <button class="btn btn-primary">PLACE ORDER</button>
+                    </div>
+                 
+                    </div>
+               
+        </div>
+    
+    </section>
```
Comment (theVivek-kumar) [inline cart.html]: remove extra spacing

File: components/cart.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    height:35rem ;
+    margin:0 3rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+}
+
+
+.cart-container .cart-box{
+width: 80%;
+margin: 3rem 4rem;
+
```
Comment (theVivek-kumar) [inline components/cart.css:106]: do proper indentation

File: components/cart.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    border: 1px solid var(--primary-grey);
+}
+.cart-box h1{
+    font-size: 1.6rem;
+    margin: 1.5rem 0 1.5rem 0;
+}
+.cart-box h3, span{
+    font-size: 1.4rem;
+    font-weight: 500;
+    margin:1rem 0 1rem 0;
```
Comment (theVivek-kumar) [inline components/cart.css:118]: what this 0 represent

File: wishlist.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+
+                <div class="card-container flex-box-default ">
+                    <div class="badge-container">
+                        <img src="https://assets.ajio.com/medias/sys_master/root/hfd/h3b/15129822035998/caprese_yellow_textured_slingbag_with_short_handles.jpg"
+                            alt="product-img" class="image-responsive product-img">
+                        <div class="text-badge">
+                            <span class="badge primary-badge">New</span>
+                        </div>
+                        <span class="iconify symbol" data-icon="mdi:cards-heart-outline" data-inline="false"></span>
```
Comment (theVivek-kumar) [inline wishlist.html:124]: cards looks good

**Review by theVivek-kumar — APPROVED** [review]

**Comment by swethaogeti** [comment]
> Looks good. Can merge it. One suggestion try to you use semantic tags.

noted !! i will improve on it

**Pull request merged**